corehq/apps/app_manager/util.py

```diff
@@ -409,46 +409,83 @@ def _add_ancestors(ctype, generation):
     else:
         generations = []
 
+    subsets = [{
+        "id": generation_names[i],
+        "name": "{} ({})".format(generation_names[i], " or ".join(ctypes)) if i > 0 else base_case_type,
+        "key": "@case_type",
```

Comment:
What's the significance of this. We don't use this in the parent case xpath expression.